extensions/amazon-lambda/common-runtime/src/main/java/io/quarkus/amazon/lambda/runtime/AbstractLambdaPollLoop.java

```diff
@@ -81,7 +85,7 @@ public void run() {
                                 continue;
                             }
                             try {
-                                if (LambdaHotReplacementRecorder.enabled) {
+                                if (LambdaHotReplacementRecorder.enabled && launchMode == LaunchMode.DEVELOPMENT) {
```

Comment:
Why only in dev mode are we doing the hot replacement check?

Reply:
Continuous testing does it's hot replacement between test runs. If further changes are made this triggers a new test run, so there is no live reload.